assets/queries/ansible/aws/public_ecr_policy/test/negative.yaml

```diff
@@ -0,0 +1,12 @@
+- name: set-policy as object
+  community.aws.ecs_ecr:
+    name: needs-policy-object
+    policy:
+      Version: '2008-10-17'
+      Statement:
+        - Sid: read-only
+          Effect: Allow
+          Action:
+            - ecr:GetDownloadUrlForLayer
+            - ecr:BatchGetImage
+            - ecr:BatchCheckLayerAvailability
```

Comment:
This policy statements should be inside a json specification, because this is cloudformation syntax